3.analyze_data/UMAP_analysis/nbconverted/2.jump_UMAP.py

```diff
@@ -0,0 +1,389 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Set up UMAP coordinates
+
+# ## Import libraries
+
+# In[1]:
+
+
+import pathlib
+import pandas as pd
+import umap
+
+
+# ## Set paths and variables
+
+# In[2]:
+
+
+# Load in barcode_platemap to assign plates based on treatment type
+barcode_platemap = pd.read_csv(
+    pathlib.Path("../../reference_plate_data/barcode_platemap.csv")
+)
+
+# Create a dictionary mapping plate to treatment type
+plate_map_dict = dict(
+    zip(barcode_platemap["Assay_Plate_Barcode"], barcode_platemap["Plate_Map_Name"])
+)
+
+# load in feature selected data
+fs_df = pd.read_parquet(pathlib.Path("./data/concat_fs_norm_data_subset.parquet"))
+
+# load in mitocheck filtered data
+filter_df = pd.read_parquet(pathlib.Path("./data/concat_mitocheck_data_subset.parquet"))
+
+# load in predicted probabilites file
+prob_path = pathlib.Path("./data/all_features_probabilities.parquet")
+prob_df = pd.read_parquet(prob_path)
+
+# Mitocheck labelled dataset to use to find nuclei features and concat
+label_data_path = "https://github.com/WayScience/phenotypic_profiling_model/raw/main/0.download_data/data/labeled_data__ic.csv.gz"
```

Comment:
When you're reading from a github url, it's generally advised to load a _versioned_ url. You can find the versioned url by pressing `y` when you're looking at any GitHub file

Reply:
Just added this! Good thing to know for the future, I never knew about this at all!